### PR TITLE
Update how-to-create-and-run-a-long-running-workflow.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 1]
```diff
@@ -60,18 +38,18 @@ One of the central features of Windows Workflow Foundation (WF) is the runtime
...
+> The steps in this procedure describe how to add and configure the form manually. If desired, you can download the solution files for the tutorial and add the completed form to the project. To download the tutorial files, see [Windows Workflow Foundation (WF45) - Getting Started Tutorial](https://go.microsoft.com/fwlink/?LinkID=248976). Once the files are downloaded, right-click **NumberGuessWorkflowHost** and choose **Add Reference**. Add a reference to **System.Windows.Forms** and **System.Drawing**. These references are added automatically if you add a new form from the **Add**, **New Item** menu, but must be added manually when importing a form. Once the references are added, right-click **NumberGuessWorkflowHost** in **Solution Explorer** and choose **Add**, **Existing Item**. Browse to the `Form` folder in the project files, select **WorkflowHostForm.cs** (or **WorkflowHostForm.vb**), and click **Add**. If you choose to import the form, then you can skip down to the next section, [To add the properties and helper methods of the form](#to-add-the-properties-and-helper-methods-of-the-form).
```
Comment (Youssef1313) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]: Please, verify that the `[To add the properties and helper methods of the form]` link would work as expected.

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 2]
```diff
@@ -299,15 +277,15 @@ The steps in this section add properties and helper methods to the form class th
     Private Sub ListPersistedWorkflows()
         Using localCon As New SqlConnection(connectionString)
             Dim localCmd As String = _
-                "Select [InstanceId] from [System.Activities.DurableInstancing].[Instances] Order By [CreationTime]"
+                "SELET [InstanceId] FROM [System.Activities.DurableInstancing].[Instances] ORDER BY [CreationTime]"
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]:
```suggestion
                "SELECT [InstanceId] FROM [System.Activities.DurableInstancing].[Instances] ORDER BY [CreationTime]"
```

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]: It's working fine, thanks!

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. Changes look good. Left a suggestion for a typo that was introduced.

File: docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md [hunk 3]
```diff
@@ -508,12 +484,12 @@ The steps in this section add properties and helper methods to the form class th
     ```vb
     wfApp.Aborted = _
         Sub(e As WorkflowApplicationAbortedEventArgs)
-            UpdateStatus($"Workflow Aborted. Exception: {0e.Reason.GetType().FullName}" & vbCrLf & $"{e.Reason.Message}")
+            UpdateStatus($"Workflow Aborted. Exception: {e.Reason.GetType().FullName}{vbCrLf}{e.Reason.Message}")
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/framework/windows-workflow-foundation/how-to-create-and-run-a-long-running-workflow.md]: Looks good. Thanks!

**Review by mairaw — APPROVED** [review]
LGTM. :shipit:

**Pull request merged**